src/providers/data_provider_be.c

```diff
@@ -127,10 +127,20 @@ void be_mark_offline(struct be_ctx *ctx)
 
         offline_timeout = get_offline_timeout(ctx);
```

Comment:
For getting offline_timeout separated helper function is used. Why not to follow the same approach with offline_timeout_max?

Reply:
Sure, that's nicer. I've added get_offline_timeout_max().